chasm/test_component_test.go

```diff
@@ -107,7 +107,7 @@ func testComponentSerializedNodes() map[string]*persistencespb.ChasmNode {
 				},
 				Attributes: &persistencespb.ChasmNodeMetadata_ComponentAttributes{
 					ComponentAttributes: &persistencespb.ChasmComponentAttributes{
-						Type:  "test_component",
+						Type:  "TestLibrary.test_component",
```

Comment:
@alexshtin Is this the right place to change? I don't see `generateMapInit` setting the Type field or probably I missed it.

Reply:
because it is in #7409.